src/main/webapp/resources/js/pages/cart/components/Cart.jsx

```diff
@@ -1,23 +1,23 @@
-import React, { lazy, Suspense, useEffect, useState } from "react";
-import { connect } from "react-redux";
-import PropTypes from "prop-types";
 import { Layout } from "antd";
-import { actions } from "../../../redux/reducers/cart";
+import React, { lazy, Suspense, useState } from "react";
+import { useCountQuery } from "../../../apis/cart/cart";
 
 const CartSamples = lazy(() => import("./CartSider"));
 const CartTools = lazy(() => import("./CartTools"));
 
 const { Content } = Layout;
 
-function CartComponent({ count = 0, loadCart }) {
+export function Cart() {
   const [collapsed, setCollapsed] = useState(false);
 
-  useEffect(() => {
-    setCollapsed(count === 0);
-    if (count !== 0) {
-      loadCart();
-    }
-  }, [count]);
+  const { data: count = 0 } = useCountQuery();
+
+  // useEffect(() => {
```

Comment:
Remove commented out code if its not required here and below

Reply:
Updated in 4bf7933335a355a16f1c1a4f4eb34b66014b2d76